velox/functions/sparksql/Map.cpp

```diff
@@ -32,50 +32,42 @@
 namespace facebook::velox::functions::sparksql {
 namespace {
 
-template <TypeKind kind>
 void setKeysResultTyped(
```

Comment:
Since this function is no longer a template, 'Typed' suffix needs to be removed.

Reply:
Updated, thanks.